org.eclipse.sisu.plexus/src/main/java/org/codehaus/plexus/ContainerConfiguration.java

```diff
@@ -62,4 +62,9 @@ public interface ContainerConfiguration
     ContainerConfiguration setJSR250Lifecycle( boolean on );
 
     boolean getJSR250Lifecycle();
+
+    ContainerConfiguration setStrictClassPathScanning( boolean strictScanning );
```

Comment:
It seems this interface was anyhow much modified with Sisu Plexus in contrast to the last known version at https://github.com/codehaus-plexus/plexus-containers/blob/3b0a8d5fd086e3f74ccdeecaac9cd63e44047e91/plexus-container-default/src/main/java/org/codehaus/plexus/ContainerConfiguration.java#L22. So adding two more methods doesn't do any harm here.

Reply:
Exactly, config was had to be changed (scanning, auto wire, etc)